packages/casadi/test_casadi.py

```diff
@@ -0,0 +1,387 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["casadi"])
+def test_symbolic_variable(selenium):
+    import casadi as ca
+
+    x = ca.SX.sym("x")
+    assert isinstance(x, ca.SX)
+    assert x.name() == "x"
+
+
+@run_in_pyodide(packages=["casadi", "numpy"])
+def test_basic_casadi_function_eval(selenium):
+    import casadi as ca
+    import numpy as np
+
+    x = ca.MX.sym("x")
+    y = ca.MX.sym("y")
+    f = ca.Function("f", [x, y], [x**2 + y**2])
+
+    assert np.allclose(f(1, 2).full(), np.array([5]))
+    assert np.allclose(f(3, 4).full(), np.array([25]))
+    assert np.allclose(f(5, 6).full(), np.array([61]))
+
+
+# IPOPT not available because compiled with -DWITH_IPOPT=OFF
+# by default for size reasons. This can be changed by setting
+# -DWITH_IPOPT=ON in the future, but it requires IPOPT to be
+# built beforehand.
+# @pytest.mark.skip(reason="IPOPT not available for now")
+# @run_in_pyodide(packages=["casadi", "numpy"])
+# def test_simple_optimization(selenium):
+#     import casadi as ca
+#     import numpy as np
+
+#     opti = ca.Opti()
+#     x = opti.variable()
+#     y = opti.variable()
+#     opti.minimize((x - 1) ** 2 + (y - 2) ** 2)
+#     opti.solver("ipopt")
+#     sol = opti.solve()
+#     assert np.allclose(sol.value(x), 1.0, atol=1e-6)
+#     assert np.allclose(sol.value(y), 2.0, atol=1e-6)
```

Comment:
Can you either uncomment it or delete it? Shouldn't `skip` be good enough?

Reply:
Oh, you suggested earlier that either of those would be fine – I had kept it commented out instead of skipping it so that it doesn't show up in the logs. I can restore it to being skipped.

Reply:
Addressed in d6ee8cae72cd7bcee35bd9a42242ea0fc8a3047f.